xbmc/TextureCache.cpp

```diff
@@ -87,6 +87,9 @@ std::string CTextureCache::GetCachedImage(const std::string &image, CTextureDeta
   // lookup the item in the database
   if (GetCachedTexture(url, details))
   {
+    if (details.file.empty())
+      return "";
```

Comment:
```suggestion
      return {};
```
will call the default constructor of std::string